packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -91,6 +96,20 @@ class AnalyzeContinuously extends AnalyzeBase {
         }
       }
 
+      // Summarize dartdoc issues rather than displaying each individually
+      int membersMissingDocumentation = 0;
+      if (flutterRepo && !showDartDocIssuesIndividually) {
+        errors.removeWhere((AnalysisError error) {
+          if (error.code == 'public_member_api_docs') {
+            // https://github.com/dart-lang/linter/issues/208
+            if (isFlutterLibrary(error.file))
+              ++membersMissingDocumentation;
```

Comment:
nit: @Hixie is trying to remove usages of `++` in our code, replacing them with `+= 1`.